src/main/java/lotto/domain/Money.java

```diff
@@ -1,21 +1,33 @@
 package lotto.domain;
 
-import lotto.Constant;
-
 public class Money {
     private static final String NOT_INTEGER_ERROR_MESSAGE = "숫자가 아닙니다.";
     private static final String UNDER_MINIMUM_ERROR_MESSAGE = "1000원 미만의 금액은 입력할 수 없습니다.";
     private static final String NOT_QUANTIZED_ERROR_MESSAGE = "입력 금액은 1000원 단위여야 합니다.";
-    private final int UNIT_PRICE;
+    private static final int UNIT_PRICE = 1000;
+    private final int initialMoney;
     private int money;
```

Comment:
상수와 변수 사이에는 한 줄 띄워주면 읽는 사람이 편하게 읽을 수 있답니다!

```suggestion
    private static final int UNIT_PRICE = 1000;
    
    private final int initialMoney;
    private int money;
```